### PR TITLE
Wfs getfeatures offline crash backport

### PR DESCRIPTION
**Pull request by elpaso**

### CODE REVIEW

File: ci/travis/linux/install.sh [hunk 1]
```diff
@@ -1,6 +1,12 @@
 mkdir build
 cd build
 
+export CXX="clang++-3.8"
+export CC="clang-3.8"
```
Comment (m-kuhn) [inline ci/travis/linux/install.sh]:
```
export CXX="clang++-3.6"
export CC="clang-3.6"
```

**Comment by elpaso** [comment]
Merged from another updated branch

**Comment by m-kuhn** [comment]
What was required to get the sip bindings built?

**Comment by elpaso** [comment]
I've no idea, I removed all travis related commits and backported only the bugfix part https://github.com/qgis/QGIS/commits/release-2_14
Still failing though: https://travis-ci.org/qgis/QGIS/builds/124959186

**Comment by m-kuhn** [comment]
> I've no idea, I removed all travis related commits and backported only the bugfix part https://github.com/qgis/QGIS/commits/release-2_14

Strange

> Still failing though: https://travis-ci.org/qgis/QGIS/builds/124959186

That shouldn't be related to your changes. Too much troubles (many different issues, internal and external) on travis the last days. I hope that it runs stable again now.

**Comment by elpaso** [comment]
Thank you for taking care of the travis stuff, I think it's something in between black magic woodoo and fingers crossed.

**Comment by dakcarto** [comment]
@m-kuhn Travis seems to be OK now on `release-2_14` (at least for Linux) with your latest commit. Thanks!

**Comment by m-kuhn** [comment]
I didn't do anything, the sip build crashes are still a miracle to me. In the past they could be resolved with clang version changes but here they disappeared for apparently no reason.

My latest commit just disabled the osx tests becuase of sip/pyqt/qscintilla deprecation and sip version issue.

**Pull request closed without merging**